b2/resource_b2_bucket.go

```diff
@@ -155,7 +155,8 @@ func resourceB2BucketRead(ctx context.Context, d *schema.ResourceData, meta inte
 	const op = RESOURCE_READ
 
 	input := map[string]interface{}{
-		"bucket_id": d.Id(),
+		"bucket_id":  d.Id(),
+		"cors_rules": d.Get("cors_rules").([]interface{}),
```

Comment:
AFAIK, you don't need the casting here, since the value is stored as `interface{}` anyway (inside a `map[string]interface{}`).

```suggestion
		"cors_rules": d.Get("cors_rules"),
```